plugin-release.js

```diff
@@ -85,75 +88,153 @@ module.exports = {
+          if (exitCode !== 0) {
+            this.context.stderr.write(`Error: git commit failed with code: ${exitCode}\n`);
+            return;
+          } else {
+            this.context.stdout.write("git tag OK!\n");
+          } 
           // Step 6 - git push the updates.
-          this.context.stdout.write(`6. git push the version update...\n`);
-          this.context.stdout.write(`6.1. git pushing the branch...\n`);
-          await execute('git', ['push'], executeOptions);
-          this.context.stdout.write(`6.2. git pushing the tag...\n`);
-          await execute('git', ['push', '--tags'], executeOptions);
-
+          this.context.stdout.write("6. git push the version update...\n");
+          this.context.stdout.write("6.1. git pushing the branch...\n");
+          exitCode = await execute("git", ["push"], executeOptions);
+          if (exitCode !== 0) {
+            this.context.stderr.write(`Error: git push failed with code: ${exitCode}\n`)
+            return;
+          } else {
+            this.context.stdout.write("git push OK!\n");
+          } 
+          this.context.stdout.write("6.2. git pushing the tag...\n");
+          exitCode = await execute("git", ["push", "--tags"], executeOptions);
+          if (exitCode !== 0) {
+            this.context.stderr.write("Error: git push tags failed with code: ${exitCode}\n");
+            return;
+          } else {
+            this.context.stdout.write("git push tags OK!\n");
+          } 
           // Step 7 - Publish the packages to npm.js.
-          this.context.stdout.write(`7. Running \`yarn npn publish\` to publish packages to npm.js...\n`);
-          await this.cli.run(['npm', 'login']);
+          this.context.stdout.write("7. Running \`yarn npn publish\` to publish packages to npm.js...\n");
+          exitCode = await this.cli.run(["npm", "login"]);
+          if (exitCode !== 0) {
+            this.context.stderr.write("Error: npm login failed with code: ${exitCode}\n");
+            return;
+          } else {
+            this.context.stdout.write("npm login OK!\n");
+          }
           for (let i = 0; i < project.workspaces.length; i++) {
-            let projectWorkspace = project.workspaces[i];
+            const projectWorkspace = project.workspaces[i];
 
             if (projectWorkspace.cwd == project.cwd) {
               this.context.stdout.write("NOTE: Skipping publish of project root workspace!\n")
             } else {
-              // Make a copy of the LICENSE file to the workspace so that it is published as part of the package
-              await copyFile(`${project.cwd}/LICENSE`, `${projectWorkspace.cwd}/LICENSE`);
-
+              const projectWorkspaceName = `@${projectWorkspace.manifest.name.scope}/${projectWorkspace.manifest.name.name}`;
+
+              const licenseFileExists = await access(`${projectWorkspace.cwd}/LICENSE`).then(x => true).catch(e => false);
```

Comment:
`then(x => true).catch(e => false)` is causing following error: 

`'x' is defined but never used. Allowed unused args must match /^_/u.eslint[@typescript-eslint/no-unused-vars](https://typescript-eslint.io/rules/no-unused-vars)`


I would recommend to prefix the variables with an underscore to mark them as optional 
`then(_x => true).catch(_e => false)`, or to set the TSconfig compilerOptions to not check JS modules.